app/components/shared/gcse_qualification_cards_component.html.erb

```diff
@@ -11,7 +11,7 @@
             <h4 class="govuk-heading-s govuk-!-margin-bottom-1"><%= subject(qualification) %></h4>
             <dl class="app-qualification">
               <dd class="app-qualification__value"> <%= candidate_does_not_have %> </dd>
-              <dt class="app-qualification__key">Reason given</dt>
+              <dt class="app-qualification__key">Details of qualification currently studying for</dt>
               <dd class="app-qualification__value"><%= qualification.not_completed_explanation || qualification.missing_explanation %></dd>
```

Comment:
This label needs to be conditional upon whether the candidate answered "Yes" or "No" to "Are you currently studying for this qualification?"

If they answered Yes, and then completed the free text, then the label should be "Details of qualification currently studying for". If they answered No, and then fill out the "If you have other evidence of having maths skills at the required standard, give details (optional)" then we should display "Other evidence I have the skills required" with their answer.

Currently this field can be mis-labelled:

<img width="378" alt="Screenshot 2022-11-24 at 10 31 48" src="https://user-images.githubusercontent.com/30665/203761916-ebe5dadf-f125-420e-b0ba-708421a363b8.png">

Hope that makes sense?